samples/client/petstore/kotlin-nonpublic/bin/main/org/openapitools/client/apis/StoreApi.kt

```diff
@@ -0,0 +1,192 @@
+/**
```

Comment:
I guess the kotlin files are unrelated to this PR

Reply:
same as the groovy